### DISCUSSION TITLE
Some change proposals

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1221] [Team Posts]
Hi guys,

I've been collecting a list of rather minor changes, here's a few that I thought merited discussion.

- Currently we are preferring knobs to sliders, for instance in the channel strip, I think we should reverse that. The slider has many benefits and looks cool.

- Also it recently dawned on me that the softsynth part of the midi port dialog is rather pointless, there's nothing to do (actually you can instantiate synths there but that is completely irregular behaviour) and it just confuses the purpose of the dialog. We should remove that part.

- A third one that I just wanted to throw up there (not convinced we should change it) is the use of the master track. We allow to disable it but it is enabled by default. I can see benefits with having it off by default mainly with regards to erroneous tempo changes (as most songs have constant tempo) and the confusion that follows.

**Comment by kybos** [comment]
>     Currently we are preferring knobs to sliders, for instance in the channel strip, I think we should reverse that. The slider has many benefits and looks cool.

Matter of taste to some extent, but the sliders give the mixer better symmetry and take less vertical space, so I agree with you.

>     Also it recently dawned on me that the softsynth part of the midi port dialog is rather pointless, there's nothing to do (actually you can instantiate synths there but that is completely irregular behaviour) and it just confuses the purpose of the dialog. We should remove that part.

Absolutely. I remember how confusing I found this dialog myself...
There are more issues: It took me weeks before I accidentally discovered that the LED-indicators are in fact checkboxes and can be clicked! And in the Port assignment list I still almost always click the wrong column, because the dropdown-arrows are on the wrong side of the text.
BTW, I already started to fix some of these issues. There are more dialogs (and also menus) with similar problems.

>     A third one that I just wanted to throw up there (not convinced we should change it) is the use of the master track. We allow to disable it but it is enabled by default. I can see benefits with having it off by default mainly with regards to erroneous tempo changes (as most songs have constant tempo) and the confusion that follows.

On the other side, then the user could do something in the mastertrack editors and see (or hear) no results - could be confusing too? But I have no strong opinion on that one...
